src/app.js

```diff
@@ -1,21 +1,23 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
 
-module.exports = sum;
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const parts = normalizedURL.pathname.slice(1).split('/');
+  const query = Object.fromEntries(normalizedURL.searchParams.entries());
```

Comment:
Cool! But if we have URL like that `/hello/world/123?x=1&search=some1&search=some2` your code will no working.
Good idea to use array for this situation - in this case output muse be an array of params, like:

```js
{
  "parts": ["hello", "world", "123"],
  "query": {
    "x": "1",
    "search": ["some1", "some2"]
  }
}
```
Think how we can implement this logic for future.